base/access_provider/sync_to_target/model.go

```diff
@@ -41,11 +41,26 @@ type Access struct {
 }
 
 type WhoItem struct {
-	Users          []string `yaml:"users" json:"users"`
-	Groups         []string `yaml:"groups" json:"groups"`
-	InheritFrom    []string `yaml:"inheritFrom" json:"inheritFrom"`
-	UsersInGroups  []string `yaml:"usersInGroups" json:"usersInGroups"`
-	UsersInherited []string `yaml:"usersInherited" json:"usersInherited"`
+	// Users contains all account names of users assigned to this access provider
+	Users []string `yaml:"users,omitempty" json:"users,omitempty"`
+
+	// Groups contains all account names of groups assigned to this access provider
+	Groups []string `yaml:"groups,omitempty" json:"groups,omitempty"`
+
+	// InheritFrom contains all access providers actual names in WHO part of this access provider
+	InheritFrom []string `yaml:"inheritFrom,omitempty" json:"inheritFrom,omitempty"`
+
+	// UsersInGroup contains all account names of users assigned to groups of this access provider
+	UsersInGroups []string `yaml:"usersInGroups,omitempty" json:"usersInGroups,omitempty"`
+
+	// UsersInherited contains all account names of users that are in an unpacked who part of a child access provider
+	UsersInherited []string `yaml:"usersInherited,omitempty" json:"usersInherited,omitempty"`
+
+	// NativeGroupsInherited contains all groups associated with the native IS that are in who part of this or a child access provider
+	NativeGroupsInherited []string `yaml:"nativeGroupsInherited,omitempty" json:"nativeGroupsInherited,omitempty"`
+
+	// UsersInheritedNativeGroupsExcluded contains all account names of users that are in who part of this or a child access provider but native groups are excluded
```

Comment:
Not 'child' but 'descendant' I suppose

With 'all', you mean unpacked? So it unpacks all groups, but not groups in the native identity store?

Reply:
That is correct